File: samples/algorithms/simple-grover/host.py
## Copyright (c) Microsoft Corporation. All rights reserved.
## Licensed under the MIT License.

# This Python script calls the ApplyGrover Q# operation
# defined in the SimpleGrover.qs file.

# For instructions on how to install the qsharp package,
# see: https://docs.microsoft.com/quantum/install-guide/python
import qsharp
from Microsoft.Quantum.Samples.SimpleGrover import SearchForMarkedInput

n_qubits = 5
result = SearchForMarkedInput.simulate(nQubits=n_qubits)
print(result)

File: samples/algorithms/simple-grover/Host.cs
// Copyright (c) Microsoft Corporation. All rights reserved.
// Licensed under the MIT License.

using Microsoft.Quantum.Simulation.Simulators;
using System;

namespace Microsoft.Quantum.Samples.SimpleGrover
{
    class Program
    {
        static void Main(string[] args)
        {
            var sim = new QuantumSimulator();
            var nDatabaseQubits = 5;

            var result = SearchForMarkedInput.Run(sim, nDatabaseQubits).Result;
            Console.WriteLine($"Result: {result}");

            Console.WriteLine("\n\nPress any key to continue...\n\n");
            Console.ReadKey();
        }
    }
}
